packages/aws-cdk-lib/aws-ecs/lib/container-image.ts

```diff
@@ -18,6 +18,8 @@ export abstract class ContainerImage {
 
   /**
    * Reference an image in an ECR repository
+   * 
```

Comment:
```suggestion
   *
```
Good thing the linter caught this space, who knows what could've happened otherwise!